apiserver/apiserver.go

```diff
@@ -44,8 +45,11 @@ func NewAPIServer(o *Options) (*APIServer, error) {
 // ListenAndServe wraps http.Server and provides context-based cancelation.
 func (api *APIServer) ListenAndServe(ctx context.Context) error {
 	svr := http.Server{
-		Addr:    api.Options.Addr,
-		Handler: api.NewRouter(ctx),
+		Addr:         api.Options.Addr,
+		Handler:      api.NewRouter(ctx),
+		ReadTimeout:  5 * time.Second,   //nolint:gomnd // variable for 5 is not needed.
```

Comment:
where exactly did these values come from?

Reply:
It was Ace's suggestion from - https://blog.cloudflare.com/exposing-go-on-the-internet/ where he thought idle was a bit high so I changed it to 100